infrastructure/k8s-config/clusters/kit-infrastructure/perfdash/perfdash.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: perfdash
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: perfdash-config
+  namespace: perfdash
+  labels:
+    app.kubernetes.io/name: perfdash
+    app.kubernetes.io/component: config
+    app.kubernetes.io/part-of: perfdash
+data:
+  jobs.yaml: |
```

Comment:
- This data about jobs are more specific to internal things that we run. It doesn't make sense to keep this in upstream repo. Could we push this in our internal repo.
wdyt ?

Reply:
That makes sense.